### ISSUE TITLE
Celestica DX010  32x100G

### ISSUE THREAD

**Issue by willieken559099** [Azure/sonic-buildimage#5162]
How to configure Celestica seastone DX010 32x100G ports to use 10G in the BCM file ?

**Comment by Mike-Ubezzi-MSFT** [comment]
@lguohan - Would it be possible to assist this this, as the same issue was raise in the MicrosoftDocs channel but there is no direct support for this topic. I am hoping that by bringing attention to the issue raised here, I can help the customer find the information they are seeking. Thank you, Mike.

**Comment by willieken559099** [comment]
Please try to help , it's related to my work

**Comment by willieken559099** [comment]
Any updates ?

**Comment by Mike-Ubezzi-MSFT** [comment]
@willieken559099 I am working with a Networking team to figure this out....apologies for the delay.

**Comment by willieken559099** [comment]
Thank you Michael

**Comment by xinliu-seattle** [comment]
sonicproject@googlegroups.com is a better place to ask this question.

**Comment by xinliu-seattle** [comment]
@zhenggen-xu is this supported by dynamic port breakout?

**Comment by willieken559099** [comment]
Yes it's , the problem that the port doesn't light up when we change the speed to 10g and I believe that has to be done in the Bcm file first, that's why I am asking how to modify the Bcm file to use 10g

**Comment by zhenggen-xu** [comment]
OK, I don't think i have the context about your goals.  What exactly you are trying to achieve overall, just change one port to a different speed?  and What sonic version was used in your testing, and which hwsku? 

Before the dynamic port breakout feature (which was onboarded recently on master),  to change the port breakout mode, e,g, from 32x100G to 96x25G/10G+16x50G,  you will have to use different hwsku.  With this PR: https://github.com/Azure/sonic-buildimage/pull/4235 on master and latest code, you could change the port speed freely from 100G, 50G, 25G/10G etc on DX010.  If it is not something you want to dynamically change all the time,  I suggest to pick a right hwsku and start from there, where we have different bcmfiles predefined for each hwsku.
